src/data/data.js

```diff
@@ -104,7 +92,51 @@ export const eventPhotos = [{
   image: 'https://lh3.googleusercontent.com/pw/AP1GczNcaSw3Sh12JA8gYo0eTp6REZyoogQJPdhyIsbx3m3tb4MpG_jC-gG-r3h8YU65idL3U8qruJeuIw4cP-JVo41f8kLJYSTFncp8iAO_vbtnl7CX2MYzNzppvfedVXC5II8kNGfrNUV5XOHw8LATa7V4=w1401-h934-s-no-gm?authuser=0',
 }];
 
-export const speakers2024 = [];
+export const speakers2024 = [{
```

Comment:
We need to create a dedicate collection for sessions like
```
export const sessions = [{
  uid: '', <-- this will be the key we need to add to speaker data
  title: '',
  description: ''',
  time: '',
  room: '',
}]
```

And add the uid to speaker data so that we can display the session in the speaker page.

Reply:
```
export const sessions2024 = [
  {
    title: 'Advanced Pregnancy Risk Assessment Using AI Model Chain',
    uuid: '789c0123-d456-f789-0123-456789abcdef', 
    description: 'This talk explores the use of AI model chains to predict pregnancy risks...',
    startTime: '2024-10-20T10:00:00', 
    endTime: '2024-10-20T11:00:00',
    room: 'Main Hall', 
    speakerUUIDs: ['123e4567-e89b-12d3-a456-426614174000'], 
  },
  
  ```
Structure for Session2024. Added in data.js